### ISSUE TITLE
System.IO.FileSystem.Tests.FileInfo_get_LastWriteTime.WriteTimeRefreshes failed

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#2359]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx_linux_debug_tst_prtest/1161/console
```
System.IO.FileSystem.Tests.FileInfo_get_LastWriteTime.WriteTimeRefreshes [FAIL]
15:18:01       Assert.InRange() Failure
15:18:01       Range:  (635725090770029800 - 635725090780030360)
15:18:01       Actual: 635725090770000000
15:18:01       Stack Trace:
15:18:01 Skipped test. Make sure you have a parent directory (not root) with somes files before running this test
15:18:01             at System.IO.FileSystem.Tests.FileInfo_get_LastWriteTime.WriteTimeRefreshes()
```

**Comment by ianhays** [comment]
Hmm. I ran the tests 100 times on my Unix machine and couldn't get them to fail, so I must be missing something.

The code will retrieve the beforeTime-1 second, then write to the file, then get the afterTime. The 1 second beforeTime buffer should resolve the Linux precision issue unless:
- There is rounding going on in DateTime.Now that is messing with my buffer
- Commands continue iterating even though IO isn't finished writing, causing afterTime to be retrieved before the file write is done.

**Comment by ianhays** [comment]
The issue was this one:

"This method may return an inaccurate value, because it uses native functions whose values may not be continuously updated by the operating system. Each operating system manages the last write time according to its own rules. To improve performance, an operating system might not set the last write time value to the exact time of the last write operation, but might set it to a close approximation instead."

The old tests had huge ranges to prevent test failures, but weren't actually <i>testing</i> file modification itself, so I've removed them in #2390 and replaced them with tests that aren't time sensitive.

**Comment by ianhays** [comment]
Closed with #2433
